cmd/habitat/community/state/raft.go

```diff
@@ -127,5 +127,4 @@ func (s *FSMSnapshot) Persist(sink raft.SnapshotSink) error {
 
 // Release is invoked when we are finished with the snapshot.
 func (s *FSMSnapshot) Release() {
```

Comment:
why is this empty?

Reply:
`FSMSnapshot` needs to fufill the `raft.FSM` interface, but currently we don't have any behavior defined for it